syntax_and_semantics/blocks_and_procs.md

```diff
@@ -163,39 +163,58 @@ The block variable `second` also includes the `Nil` type because the last `yield
 
 ## Short one-argument syntax
 
-A short syntax exists for specifying a block that receives a single argument and invokes a method on it. This:
+If a block has a single argument and invokes a method on it, the block can be replaced with the short syntax argument.
+
+This:
 
 ```crystal
 method do |argument|
   argument.some_method
 end
 ```
 
-Can be written as this:
+and
+
+```crystal
+method { |argument| argument.some_method }
+```
+
+Can both be written as:
```

Comment:
"This", the code block, the "and", the other code block and these words are still one sentence so this shouldn't be capitalized.
```suggestion
can both be written as:
```